src/Compilers/Core/CodeAnalysisTest/ObjectSerializationTests.cs

```diff
@@ -1197,13 +1233,20 @@ private class Node : IObjectWritable, IEquatable<Node>
         {
             internal readonly string Name;
             internal readonly Node[] Children;
+            private readonly bool _isReusable = true;
 
             public Node(string name, params Node[] children)
             {
                 this.Name = name;
                 this.Children = children;
             }
 
+            public Node(string name, bool isReusable)
```

Comment:
note is a pretty terrible name :)   i was very confused here :)

Reply:
It was `Node` before. I will leave it as-is.

---
In reply to: [181531414](https://github.com/dotnet/roslyn/pull/26156#discussion_r181531414) [](ancestors = 181531414)